mobile/cmp/panel/DialogPanel.scss

```diff
@@ -23,6 +23,11 @@
       & > .xh-panel {
         width: 100%;
         height: 100%;
+
+        // Additional padding on dialog panel's own header.
+        > .xh-panel-header {
+          padding: 10px;
+        }
```

Comment:
I could have gone further and added a CSS var but got tired, and we don't have a super good story with mobile-only vars.

This gives full-screen dialogs a bit more padding / height on their top-level header - it looked weirdly thin to me without